test/e2e/autoscaling_utils.go

```diff
@@ -147,3 +170,30 @@ func startService(c *client.Client, ns, name string, replicas int) {
 	}
 	expectNoError(RunRC(config))
 }
+
+func createHorizontalPodAutoscaler(name, ns, cpu string) *client.ExperimentalClient {
+	config, err := loadConfig()
+	expectNoError(err)
+	expClient, errExpClient := client.NewExperimental(config)
+	expectNoError(errExpClient)
```

Comment:
After recent changes ExperimentalClient is already available as a part of regular Client. Just call rc.framework.Client.Experimental().